### DISCUSSION TITLE
Tailwind css not working after export build in nextjs

### DISCUSSION THREAD

**Discussion by studentofmusk** [vercel/next.js#52212] [App Router] [answered]
Recently, I switch to nextjs app router version
I installed and added all neccessary thing to use railwind css in nextjs project.
It works good while development.

But if i generate static build, my tailwind css classes are not working.

I added next build && next export 
For build generation

**Comment by icyJoseph** [comment]
Hi, which Next.js version are you using? (it works for me in a recent test I just made)

**Reply by studentofmusk** [reply]
Latest only

**Reply by studentofmusk** [reply]
Okay, can you please tell me the right way to generate static page in nextjs

**Reply by icyJoseph** [reply]
I did a `yarn create next-app some-project-name`, then in the `next.config.js` file I added `output: 'export'`, and then I ran `yarn build`, and that was it.

Could you share some code or setup files, like `next.config.js`, `tailwind.config.js` and the others? Maybe you are not telling Tailwind to read the `app` router directory?

```js
/** @type {import('tailwindcss').Config} */
module.exports = {
  content: [
    './src/pages/**/*.{js,ts,jsx,tsx,mdx}',
    './src/components/**/*.{js,ts,jsx,tsx,mdx}',
    './src/app/**/*.{js,ts,jsx,tsx,mdx}',
  ],
  theme: {},
  plugins: [],
}
```

It should look something like that.

**Reply by studentofmusk** [reply]
Wait I'll try it once

**Reply by studentofmusk** [reply]
I'm actually doing the same 
But additionally `package.json` added `"build" : "next build && next export"`

**Reply by studentofmusk** [reply]
Is this make any issue?

**Reply by studentofmusk** [reply, marked as answer]
I've got solution !
`yarn start` isn't work for  new versions
Then i used `npx serve@latest out`
Now, I can see my site with tailwind css styles

**Reply by icyJoseph** [reply]
When you export as static HTML, next start is not an option. You just needed a way to serve the files, such as the serve package. I assumed you were doing that already, my bad.

**Reply by studentofmusk** [reply]
I have another problem too
While open directly the index.html file from the build, it isn't have any tailwindcss style 
Eventhit have class name
Is that make any issue while deployment

**Reply by icyJoseph** [reply]
If you are using `export` model, you get back a set of static HTML files, you need a static assets server to handle that. 

For example, locally you used `serve`. If you want to deploy your app, it has to be on a static server host. Where are you deploying?

**Reply by studentofmusk** [reply]
Yes, It works good.
Thank you 

I'm just working with a project and I stucked at the end.

Thanks again Mr.Josph

**Reply by samuel1544** [reply]
Please how did you succeed to make you style appear on your static pages?
What's your solution
